package.json

```diff
@@ -2,55 +2,43 @@
   "name": "@emscripten-forge/untarjs",
   "version": "1.0.0",
   "description": "A simple project to unzip .zip, .tar.bz2 and .conda files in the front-end",
-  "main": "dist/bundle.js",
-  "files": [
-    "dist/*.{d.ts,js,wasm,txt}"
-  ],
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
   "scripts": {
     "build:wasm": "node build_wasm.js",
     "clean": "rimraf dist",
-    "build": "yarn run clean && yarn run build:wasm && cross-env NODE_ENV=development webpack --mode development",
-    "build:prod": "yarn run clean && yarn run build:wasm && webpack",
-    "start": "cross-env NODE_ENV=development webpack serve --mode development"
+    "build": "yarn run build:wasm && tsc",
+    "build:prod": "tsc && yarn run clean && yarn run build:wasm"
   },
+  "files": [
+    "lib/index.js",
+    "lib/index.d.ts",
+    "lib/helper.js",
+    "lib/helper.ts",
+    "lib/unpack.wasm",
+    "lib/unpack.js",
+    "lib/unpack.d.ts",
+    "lib/unpack.wasm.d.ts"
+  ],
```

Comment:
can we use wildcards here?

Reply:
Hello @trungleduc We cannot use directly wildcards here, but I suggest the next solution: ```
"files": [
    "lib/"
  ],
```

Reply:
hmm, i think we could https://github.com/jupyterlab/jupyterlab/blob/090fe7383422b59588f0337fe0b5ee0d74899204/packages/application/package.json#L26

But if your solution works then it's ok

Reply:
Thank you @trungleduc for the example. Yes, my approach works too.